index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+var semver = require('semver');
+var execa = require('execa');
+var del = require('del');
+
+var exec = function (cmd, args) {
+	const result = execa.sync(cmd, args);
+
+	if (result.stderr !== '') {
```

Comment:
Alright. Just use console.log/err then.

Do you get the error even with the latest master of `execa`? If so, would you mind opening an issue? `stdio` should definitely be supported.